src/main/java/com/github/kaktushose/jda/commands/dispatching/reply/GlobalReplyConfig.java

```diff
@@ -0,0 +1,31 @@
+package com.github.kaktushose.jda.commands.dispatching.reply;
+
+import com.github.kaktushose.jda.commands.annotations.interactions.ReplyConfig;
+
+/// Will be used if no [ReplyConfig] annotation is present.
+public class GlobalReplyConfig {
```

Comment:
We have to introduce this as a user api.

Reply:
Yes but no, we can leave this as it is until we introduce the jda-commands builder